app/frontend/packs/nationalities-component.js

```diff
@@ -0,0 +1,118 @@
+const nationalitiesComponent = () => {
+  if (
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality2-field-error]"
+    ) ||
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality3-field-error]"
+    )
+  ) {
+    return;
+  }
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  addAddNationalityButton(
+    "#candidate-interface-nationalities-form-other-other-conditional"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  function addRemoveLink(selector) {
+    const labelEl = document.querySelector(`[for=${selector}]`);
+    const removeLink = document.createElement("a");
+    removeLink.innerHTML = "Remove";
+    removeLink.classList.add("govuk-link", "remove-link");
```

Comment:
Custom CSS should be prefixed with `app-`:

```suggestion
    removeLink.classList.add("govuk-link", "app-remove-link");
```

Going to add a suggestion to the CSS file as well.